Source/UIViewController+SemiModalViewController.swift

```diff
@@ -169,7 +170,14 @@ extension UIViewController {
     
     @objc func interfaceOrientationDidChange(_ notification: Notification) {
         guard let overlay = parentTargetView().viewWithTag(semiModalOverlayTag) else { return }
-        addOrUpdateParentScreenshotInView(overlay)
+        let view = addOrUpdateParentScreenshotInView(overlay)
+        view.alpha = CGFloat(self.optionForKey(.parentAlpha) as! Double)
+        
+        let transitionStyle = optionForKey(.transitionStyle) as! SemiModalTransitionStyle
+        
+        if transitionStyle == .fadeIn || transitionStyle == .fadeOut {
+            overlay.alpha = 0
```

Comment:
`transitionStyle` is for SemiView, we don't need to set `alpha` for `overlayView` here.

Reply:
fix it